http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/DefaultNodepoolConfig.scala

```diff
@@ -0,0 +1,6 @@
+package org.broadinstitute.dsde.workbench.leonardo.config
+
+import org.broadinstitute.dsde.workbench.google2.MachineTypeName
+import org.broadinstitute.dsde.workbench.leonardo.NumNodes
+
+case class DefaultNodepoolConfig(machineType: MachineTypeName, numNodes: NumNodes, autoscalingEnabled: Boolean)
```

Comment:
kinda prefer to have these configs in one file so that it's easier to see relationships between them if any